src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -2038,6 +2038,63 @@ paths:
       - jwt: []
       tags:
       - Cephfs
+  /api/cluster:
+    get:
+      parameters: []
+      responses:
+        '200':
+          content:
+            application/vnd.ceph.api.v1.0+json:
```

Comment:
If we are pretty confident that these new endpoints are stable then it's OK.
Otherwise, let's mark them as experimental.

Reply:
Totally agree... Now we have more experience with 'immature' endpoints, it'd probably better to start creating all new endpoints with experimental version, and only switch to an stable after some time has passed.

Reply:
Done